app/scripts/controllers/meeting.status.ctrl.js

```diff
@@ -78,6 +78,42 @@ angular.module('dashboard')
             }
         }
 
+        function topicEdited(event, mtgGuid, mtg) {
+            if (!angular.isObject(event) || !angular.isObject(event.topic) || !angular.isObject(mtg) || !mtgGuid || !angular.isArray(mtg.topicList)) {
+                $log.error("meetingStatusCtrl.topicEdited: ignored, bad args");
+                return;
+            }
+
```

Comment:
- 'mtgGuid' above can be replaced by '!angular.equals(event.meetingID, mtgGuid)' below.

- '!angular.equals(event.meetingID, mtgGuid)' below can be removed.